app/src/main/java/com/epfl/beatlink/ui/library/Library.kt

```diff
@@ -1,4 +1,4 @@
-package com.epfl.beatlink.ui.library
+package com.android.sample.ui.library
```

Comment:
The import doesn't match the package structure.

```suggestion
package com.epfl.beatlink.ui.library
```